src/Hng.Graphql/ConfigureGraphql.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Hng.Graphql.Features;
+using Hng.Graphql.Features.Roles;
+using Hng.Graphql.Features.UserManagement;
+using HotChocolate.Execution.Configuration;
+
+namespace Hng.Graphql
+{
+    public static class ConfigureGraphql
+    {
+        public static IServiceCollection RegisterFeatures(this IServiceCollection services)
+        {
+            services
+                .AddScoped<UserManagementQueries>()
```

Comment:
Okay... This is wrong. If we start doing this, we will have a lot of Types...
We only need two type called `Queries` and `Mutations`. All queries goes into the query type and all mutations goes into the mutation type.